distributed/tests/test_actor.py

```diff
@@ -618,6 +628,42 @@ def worker_tasks_running(dask_worker):
 
 
 def test_one_thread_deadlock():
+    class UsesCounter:
+        # An actor whose method argument is another actor
+
+        def do_inc(self, ac):
+            return ac.increment().result()
+
+    with cluster(nworkers=2) as (cl, w):
+        client = Client(cl["address"])
+        ac = client.submit(Counter, actor=True).result()
+        ac2 = client.submit(UsesCounter, actor=True, workers=[ac._address]).result()
+
+        assert ac2.do_inc(ac).result() == 1
+
+
+def test_one_thread_deadlock_timeout():
+    class UsesCounter:
+        # An actor whose method argument is another actor
+
+        def do_inc(self, ac):
+            return ac.increment().result(timeout=1)
```

Comment:
I'm wondering if this shouldn't be larger (e.g. 10) just to be robust on CI. The test logic should not be impacted by this.

Reply:
in this test, `ac.increment()` returns an `EagerActorFuture` and so the timeout cannot expire

Reply:
Instead of increasing the timeout I lowered the it to `0.0001` to show that the timeout cannot expire. I also added a comment